### DISCUSSION TITLE
Publishers for CA

### DISCUSSION THREAD

**Discussion by jjmasdeu** [Keyfactor/ejbca-ce#62] [Q&A]
Hi! 
  Is there a way to configure a publisher to act as CA publisher not to certificate profile publisher?, to publish every cert profiles of a whole CA? I see I can configure publisher in CA configs but just for CRL. If I have multiple subCA using same cert profiles, then when I issue a cert for a profile "A" then it will publish on every publisher configured for that profile, so I must generate a profile with its own publisher for EVERY CA.

For example, if Ia have CA "A", "B" and "C" and one cert profile "ENDUSER" to  be used for the three AD publisher (CA "A", "B" and "C") I cannot use the same certificate profile for all CA because if I issue a cert with ENDUSER profile for CA "A" it will publish cert on all AD publisher at the same time. So, is there a way to have a  publisher based on CA and not on Certificate profile?

**Comment by primetomas** [comment]
I guess this is the same topic as this, correct?
https://sourceforge.net/p/ejbca/discussion/123122/thread/dee7fa6afc/?limit=50#055b

In that case:
---
Hi! You conclusion is correct. In order to publish certificates, your Publisher has to be selected under the relevant Certificate Profile. The Publisher selected in CA settings applies to CRL publishing only.
